From: Carl Suster <carl@contraflo.ws>
Date: Sun, 8 Jan 2017 19:28:57 +1100
Subject: disable github fork ribbon

Reconfigures documentation to exclude the GitHub fork ribbon image
hosted on the web, which triggers a privacy-breach-generic lintian
warning.

Forwarded: not-needed
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 6dd6947..cfae91b 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -105,7 +105,7 @@ html_theme = 'flask_small'
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = dict(github_fork='maxcountryman/flask-login', index_logo=False)
+html_theme_options = dict(github_fork=False, index_logo=False)
 
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path = ['_themes']
